apps/frontend/src/styles/HomePage.css

```
@@ -0,0 +1,137 @@
+  transform: translateY(14vh);
+}
+
+.hero-button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 10px;
+  padding: 10px 24px;
+  min-width: 0;
+  border-radius: 12px;
+  text-decoration: none;
```

Stylelint (17.4.0)

[error] Unexpected duplicate "text-decoration" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)